daktari/checks/intellij_idea.py

```diff
@@ -77,21 +77,17 @@ def get_intellij_idea_version_snap() -> Optional[VersionInfo]:
 
 
 def get_intellij_idea_version_tarball() -> Optional[VersionInfo]:
-    try:
-        idea_bin_path = run_command(["sh", "-c", "which idea.sh"]).stdout.rstrip("\n")
-    except CommandErrorException:
-        logging.debug("Could not locate idea.sh", exc_info=True)
+    idea_bin_path = get_stdout(["sh", "-c", "which idea.sh"])
```

Comment:
```suggestion
    idea_bin_path = get_stdout(["sh", "-c", "which", "idea.sh"])
```

☝️ wouldn't we want this?

Reply:
oh, I guess so? I don't know why we do this weird array-based approach tbh, we don't do it everywhere. But yeah, can change to that for consistency (and the other one) :shrug:

Reply:
Oh, no - making this change actually breaks it. As written, the command becomes `sh -c "which idea.sh"`. Splitting it like you've suggested turns it into `sh -c which idea.sh`, which isn't what we're intending here.